modules/ravestate_ros2/ros2_properties.py

```diff
@@ -99,11 +100,11 @@ def ros_to_ctx_callback(ctx, msg, prop_name: str):
             global_prop_set.add(prop.__hash__())
 
         # spin once
-        rclpy.spin_once(node, timeout_sec=0)
+        #rclpy.spin_once(node, timeout_sec=0)
```

Comment:
```suggestion
        rclpy.spin_once(node, timeout_sec=0)
```
node has to be spun regardless of the roboyio solution